### PR TITLE
Fix the issue of download plans (only coverpage can be downloaded)

### PR DESCRIPTION
**Pull request by pengyin-shan**
Fixes part of #153 .

This PR does not include JSON issue (unique on production and staging) and functional tests. JSON issue might be a configuration issue, will do further navigation after this emergency release. The functional test will come soon.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## [3.0.4+portage-3.0.5]
+- Fixed pdf download issue
```
Comment (lagoan) [inline CHANGELOG.md]: We need to add this on the CHANGELOG under a new version. 
Also, we need to be under a subheading for FIXED.
Now that we are keeping better track of issues we should start adding links to the issue being fixed #153 

You can see[ jupiter's CHANGELOG](https://github.com/ualbertalib/jupiter/blob/master/CHANGELOG.md) as an example

File: app/views/plans/_download_form.html.erb [hunk 2]
```diff
@@ -111,7 +111,7 @@
             options_for_select(Settings::Template::VALID_MARGIN_RANGE.to_a,
                                @export_settings.formatting[:margin][:right]),
             class: 'form-control',
-            "data-default": @plan.template.settings(:export).formatting[:margin][:rigth] %>
+            "data-default": @plan.template.settings(:export).formatting[:margin][:right] %>
```
Comment (lagoan) [inline app/views/plans/_download_form.html.erb:114]: Nice catch!

File: CHANGELOG.md (hunk 1 above)
Comment (pengyin-shan) [reply CHANGELOG.md]: Working on it : ) should I put today as the release date?

Comment (lagoan) [reply CHANGELOG.md:4]: Yes please

**Review by lagoan — CHANGES_REQUESTED** [review]
Thing are looking good. Just some minor comments made there about the change log.

We should also add a skipped stub for testing so we have a reference of where we need to work on this next.

File: CHANGELOG.md (hunk 1 above)
Comment (lagoan) [reply CHANGELOG.md]: Yes please

File: config/environments/development.rb [hunk 3]
```diff
@@ -71,4 +71,4 @@
 end
 
 # Used by Rails' routes url_helpers (typically when including a link in an email)
-Rails.application.routes.default_url_options[:host] = "localhost:3000"
+# Rails.application.routes.default_url_options[:host] = "localhost:3000"
```
Comment (lagoan) [inline config/environments/development.rb]: Is this line part of the problem for this issue?

**Review by lagoan — APPROVED** [review]
LGTM!

**Pull request merged**